Detections/CommonSecurityLog/DiscordCDNRiskyDownload.yaml

```diff
@@ -0,0 +1,50 @@
+id: 010bd98c-a6be-498c-bdcd-502308c0fdae
+name: Discord CDN Risky File Download
+description: |
+  'Identifies  callouts to Discord CDN addresses for risky file extensions. This detection will trigger when a callout for a risky file
+  is made to a discord server that has only been seen once in your environment. Unique discord servers are identified using the server ID
+  that is included in the request URL (DiscordServerId in query). Discord CDN has been used in multiple campaigns to download additional payloads'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Zscaler
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 7d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1071.001
+tags:
+  - Discord
+query: |
+  let connectionThreshold = 1;
+  let riskyExtensions = dynamic([".bin",".exe",".dll",".bin",".msi"]);
+  CommonSecurityLog
+  | where RequestURL contains "media.discordapp.net" or RequestURL contains "cdn.discordapp.com"
+  | where RequestURL has "attachments"
+  | where DeviceAction !~ "blocked"
```

Comment:
May be we can also add `DeviceVendor == "ZScaler"` to filter to limit input dataset as connectorId is limited to Zscaler

Reply:
Great point, added in